0012-common-ledger-api/0012-common-ledger-api.md

```diff
@@ -0,0 +1,1201 @@
+# Common Ledger API
+
+The Common Ledger API is a RESTful API served by a ledger (or an adapter), which provides functionality necessary for ILP compatibility. The Common Ledger API provides a single standard API that a ledger can serve in order to ease integration with other Interledger Protocol components and applications, such as the reference ILP Client and ILP Connector. This is not the only way a ledger can become ILP-enabled, but it provides a template that minimizes the integration work necessary for compatibility with ILP software.
+
+## Overview
+
+The Common Ledger API defines the following data types and structures:
+
+- [Transfer resource][]
+- [Account resource][]
```

Comment:
I don't see how you'd split it off. The whole REST API would probably be served by some "proxy" server (not as in HTTP proxy).